community-guide.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: index
+title: GCAM Community Paper Guidlines
```

Comment:
Please update to "Community guidelines for peer-reviewed journal articles using GCAM" for clarity.